account_bank_statement_clear_partner/readme/USAGE.rst

```diff
@@ -0,0 +1,6 @@
+
+To use this module, you need to:
+
+#. Go to **Invoices** > **Bank**
```

Comment:
These instructions are not correct, as that's not the correct menu path (Invoicing is the main menu, but then explained about the dashboard. You need also to explain - probably in configuration section -, that for having such dashboard, you need to enable "Show Full Accounting Features" in your user).